### PR TITLE
Regenerate cabal.project

### PR DESCRIPTION
**Pull request by hasufell**

### CODE REVIEW

**Review by KtorZ — APPROVED** [review]

**Comment by hasufell** [comment]
This should wait for https://github.com/input-output-hk/cardano-addresses/pull/81 because it isn't buildable

**Comment by rvl** [comment]
OK I see so there was some "merge skew" and the `cabal.project` regeneration step taken in #2212 did not include the changes from #2198.

Why do you say that this revision of cardano-addresses isn't buildable? It builds fine for me:
```
$ nix-shell nix/cabal-shell.nix
[nix-shell:~/iohk/cw/pr]$ cabal build --project-file=cabal.project --enable-tests --enable-benchmarks cardano-addresses
Up to date
[nix-shell:~/iohk/cw/pr]$ cabal run --project-file=cabal.project cardano-addresses-cli:exe:cardano-address -- --version
... snip ...
Linking /home/rodney/iohk/cw/pr/dist-newstyle/build/x86_64-linux/ghc-8.6.5/cardano-addresses-cli-2.1.0/x/cardano-address/build/cardano-address/cardano-address ...
2.1.0 @ e0ab4587266430a08734e1aec1c29d261a9a3b70
```

**Comment by hasufell** [comment]
> Why do you say that this revision of cardano-addresses isn't buildable? It builds fine for me:

https://github.com/input-output-hk/cardano-addresses/pull/81

**Comment by hasufell** [comment]
bors merge

File: stack.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ extra-deps:
 
 # cardano-addresses-2.1.0
 - git: https://github.com/input-output-hk/cardano-addresses
-  commit: e0ab4587266430a08734e1aec1c29d261a9a3b70
+  commit: 8ececd0d7c0ed6e0c654ff94dac8ad91a5ad596a
```
Comment (KtorZ) [inline stack.yaml:49]: That is not okay.

**Review by KtorZ — CHANGES_REQUESTED** [review]
Please revert the change in stack.yaml. 
We do only use *released* version of cardano-addresses and cardano-transactions.

File: stack.yaml (hunk 1 above)
Comment (hasufell) [reply stack.yaml:49]: Then we need a new release, because cabal users can currently not build the wallet.

**Comment by rvl** [comment]
Umm hang on @hasufell - did you read my message above?
I am able to build cardano-wallet master branch with cardano-addresses 2.1.0 (rev e0ab458) using cabal.
There was no need to use the latest master rev of cardano-addresses.
In the PR, you did not specify what actual problem it was solving.
What is the error message that you are seeing? (as in error message, not warning)

**Comment by hasufell** [comment]
>  I am able to build cardano-wallet master branch with cardano-addresses 2.1.0 (rev e0ab458) using cabal.

You are using nix. If you test with actual cabal, it doesn't work: https://github.com/input-output-hk/cardano-addresses/commit/c477127ccb1d6a2568936962bbf7c6764fa9af06#r43168725

**Comment by rvl** [comment]
> You are using nix. If you test with actual cabal, it doesn't work.

All I use Nix for is to get a shell with ghc-8.6.5, cabal-3.2.0.0 and system dependencies (reproducible environment in `nix/cabal-shell.nix`).
Then I built it with `cabal build` - actual cabal. I pasted the commands above.

This is why I'm baffled by your claims "it isn't buildable"!

**Comment by hasufell** [comment]
> This is why I'm baffled by your claims "it isn't buildable"!

Then try without nix.

**Pull request closed without merging**